js/utils.js

```diff
@@ -17,6 +17,55 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
   };
   var computedFieldClashes = [];
 
+  function setLineClamps () {
```

Comment:
Since setClampHeight supports a `selector` parameter...

```suggestion
  function setLineClamps (selector) {
```